app.js

```diff
@@ -34,6 +35,7 @@ function clientLoaded (client) {
   client.on('StasisStart', function(event, channel) {
     if (!isDialed(event.args[0])) {
       var extension = event.args[0];
+      var confFile = process.argv[2];
```

Comment:
Why would this not be extracted prior to StasisStart? As it is, it looks like you are extracting the confFile for every single channel that enters into the application. You should only be doing it once, when your client loads.